src/doom/d_main.c

```diff
@@ -2151,6 +2151,16 @@ void D_DoomMain (void)
 
     PrintGameVersion();
 
+    p = M_CheckParmWithArgs("-playmusic", 1);
+    if (p)
+    {
+        DEH_printf("Now playing %s\n", myargv[p+1]);
+        S_PlayMusicLump(myargv[p+1]);
+        while (I_MusicIsPlaying());
+        DEH_printf("Music done playing\n");
```

Comment:
You could safely exit the whole application by returning with `I_Error("Music done playing\n");`

Also, it is not necessary to use the `DEH_` prefixed string printing functions, because the strings you are going to print have never actually been in Doom.